vxquery-core/src/main/java/org/apache/vxquery/xmlparser/SAXContentHandler.java

```diff
@@ -66,6 +66,7 @@
     protected final List<ElementNodeBuilder> enbStack;
     protected final List<ElementNodeBuilder> freeENBList;
     protected boolean isIndexHandler;
+    protected boolean index;
```

Comment:
I think we can make it so there are no changes here or at least ones that say "if indexing".